src/pages/NotFoundPage/NotFoundPage.jsx

```diff
@@ -1,9 +1,9 @@
 import { Link } from 'react-router-dom';
 import { translations } from '../../utils/translations';
-import pageNotFoundIcon from '../../utils/icons/notfound.png';
 import * as S from './NotFoundPage.styles';
 
 const NotFoundPage = () => {
+  const pageNotFoundIcon = require('../../utils/icons/notfound.png');
```

Comment:
Because if I importing it like es6 syntax I have error in my test. I tried change webpack setting, but unfortunately only in this way it works without any errors.. :) 

![Error 2022-05-30 195703](https://user-images.githubusercontent.com/69632427/171384699-84ddccb3-cde0-4c94-b3ad-0c5b88ccaed4.png)

Reply:
Error fixed.